site/src/pages/DeploySettingsPage/NotificationsPage/NotificationEvents.stories.tsx

```diff
@@ -47,6 +47,8 @@ export const NoWebhookEndpoint: Story = {
 				},
 				email: {
 					smarthost: "smtp.example.com",
+					from: "localhost",
```

Comment:
Nit: it's an address

```suggestion
					from: "bob@localhost",
```